llvm/lib/Target/DirectX/DXILOpBuilder.cpp

```diff
@@ -257,7 +257,7 @@ static FunctionCallee getOrCreateDXILOpFunction(dxil::OpCode DXILOp,
   // FIXME: find the issue and report error in clang instead of check it in
   // backend.
   if ((Prop->OverloadTys & (uint16_t)Kind) == 0) {
-    llvm_unreachable("invalid overload");
+    report_fatal_error("Invalid Overload Type", false);
```

Comment:
Generally best to include a comment for boolean flag args:
```c++
    report_fatal_error("Invalid Overload Type", /*gen_crash_diag=*/false);
```

Reply:
Done.